cypress_shared/pages/temporary-accommodation/manage/bedspaceShow.ts

```diff
@@ -32,6 +35,15 @@ export default class BedspaceShowPage extends Page {
     this.shouldShowKeyAndValues('Notes', this.room.notes.split('\n'))
   }
 
+  shouldShowPremisesAttributes(): void {
```

Comment:
Maybe just put this all inside `shouldShowBedspaceDetails`, then it automatically happens anywhere we're calling that already

Reply:
yeah, I'm sort of hesitant about that so we can preserve the premises/bedspace distinction as far as possible